src/style.css

```
@@ -0,0 +1,128 @@
+body {
+  font-family: Roboto, sans-serif;
+}
```

```suggestion
html {
  font-family: Roboto, sans-serif;
}
```

i'd rather use `html` selector